api/tasks.go

```diff
@@ -108,10 +107,8 @@ func (api *API) GetTasksHandler(w http.ResponseWriter, req *http.Request) {
 	id := vars["id"]
 	logData := log.Data{"job_id": id}
 
-	paginator := pagination.NewPaginator(api.cfg.DefaultLimit, api.cfg.DefaultOffset, api.cfg.DefaultMaxLimit)
-	offset, limit, err := paginator.ValidatePaginationParameters(offsetParam, limitParam)
+	offset, limit, err := api.setUpPagination(w, offsetParam, limitParam, ctx)
 	if err != nil {
-		log.Error(ctx, "pagination validation failed", err)
```

Comment:
I think we should bring this line back now that we are not logging the error within `setUpPagination`

Reply:
yes, that makes sense - done